pkg/k8shandler/visualization.go

```diff
@@ -389,7 +390,17 @@ func isKibanaCRDDifferent(current *es.Kibana, desired *es.Kibana) bool {
 }
 
 func (clusterRequest *ClusterLoggingRequest) removeKibanaCR() error {
-	cr := newKibanaCustomResource(clusterRequest.cluster, constants.KibanaName)
+	cr := &es.Kibana{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Kibana",
+			APIVersion: es.SchemeGroupVersion.String(),
+		},
```

Comment:
You don't need to set this up, because we use client-go for the request.

Reply:
Fixed